syntax = "proto3";

package bakdata.bafin.v1;

message Bafin_Issuer {
  uint32 issuer_id = 1;
  string issuer = 2;
  string domicile = 3;
  string country = 4;
}



